pageserver/src/branches.rs

```diff
@@ -454,3 +466,42 @@ fn find_latest_snapshot(conf: &PageServerConf, timeline: ZTimelineId) -> Result<
 
     Ok((maxsnapshot, snapshotdir.unwrap()))
 }
+
+
+pub fn init_logging() -> Result<slog_scope::GlobalLoggerGuard, io::Error> {
```

Comment:
Can't we use init_logging from pageserver.rs here?

Reply:
Sure, it would be better. I'll move init_logging() into a separate module to reuse.

I wonder, why we don't redirect the log to file in a non-daemon mode? Was it intentional?
https://github.com/zenithdb/zenith/blob/main/pageserver/src/bin/pageserver.rs#L329

Reply:
Yes, that's intentional. E.g. systemd grabs stdout and manages logs by his own rules. Postgres follows this convention too.

Reply:
Hmm, but we always start pageserver with -d flag (at least if we use CLI) and do redirect logs to the file. So now this code branch is never used. Do I miss something?

Reply:
IMHO it would be nice to decouple logging settings from the daemon mode flag. @lubennikovaav since you're already working with logging, could you add another flag to control the logs destination? I imagine we could force file-based logging in a daemon mode, yet make it possible to choose between a file or a tty otherwise.